tests/test_flows.py

```diff
@@ -327,8 +391,9 @@ def test_ap_config_bss_tear_down_cli(self):
         # Same test as the previous one but using CLI instead of dev_send_1905
 
         env.beerocks_cli_command('bml_clear_wifi_credentials {}'.format(env.agents[0].mac))
-        env.beerocks_cli_command('bml_set_wifi_credentials {} {} {} {} {}'.format(env.agents[0].mac,
-                                 "Multi-AP-24G-3-cli", "maprocks1", "24g", "fronthaul"))
+        env.beerocks_cli_command('bml_set_wifi_credentials {} {} {} {} {}'
+                                 .format(env.agents[0].mac,
+                                         "Multi-AP-24G-3-cli", "maprocks1", "24g", "fronthaul"))
```

Comment:
What changed here?

Reply:
Vitaly set something in the VSCODE configuration which runs formatting automatically whenever we save a file, so it decided to format it like this. Since I changed the file in the first place, I can't avoid it.